cluster-autoscaler/cloudprovider/azure/azure_scale_set.go

```diff
@@ -767,6 +837,42 @@ func (scaleSet *ScaleSet) getOrchestrationMode() (compute.OrchestrationMode, err
 	return vmss.OrchestrationMode, nil
 }
 
-func isOperationNotAllowed(rerr *retry.Error) bool {
```

Comment:
Not present on older versions due to #6447. But seemingly justified(?) due to fork code differences, see azure_force_delete_scale_set.go for after-fork location.

This feature itself and the after-fork location is already in the fork from past versions, but #6447 upstream it in master.